lib/controller/controller.go

```diff
@@ -566,6 +595,27 @@ func (ctrl *ProvisionController) updateStats(claim *v1.PersistentVolumeClaim, er
 	}
 }
 
+func (ctrl *ProvisionController) updateDeletionVolumesStats(volume *v1.PersistentVolume, err error) {
```

Comment:
we can probably combine this with updateStats and make it take more parameters since thye look almost the same. But again, don't need to do it now/here.